lib/darjeelink/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Darjeelink
-  VERSION = '0.14.3'
+  VERSION = '0.14.5'
```

Comment:
assuming the Git tag of this version number is a separate step in the process? I can't remember, it's cool if it is :v: